### ISSUE TITLE
Implement quiz system

### ISSUE THREAD

**Issue by MunteanuAndreiStefan** [MunteanuAndreiStefan/ProjectTW#44]

**Comment by MunteanuAndreiStefan** [comment]
Status update?

**Comment by MunteanuAndreiStefan** [comment]
2 days ago you changed the deadline when I asked about a status update. When is going to be ready?

**Comment by ilikehaskell** [comment]
I need more time for this. It should be done until Sunday. Sketched out a frontend and a database.

**Comment by MunteanuAndreiStefan** [comment]
Where is the implementation? @ilikehaskell

**Comment by ilikehaskell** [comment]
i work on a separate project

**Comment by MunteanuAndreiStefan** [comment]
Any update? At least a valid frontend for the 3 types of users?

**Comment by ilikehaskell** [comment]
yes, valid front end core done for all classes, working on teacher view, I'll push on main after I validate html and css files.
